.flake8

```diff
@@ -20,6 +20,6 @@ ignore = F401, F403, F405, F811, E301, E302, E305, E501, E701, E704, E741, B303,
 # We are checking with Python 3 but many of the stubs are Python 2 stubs.
 # A nice future improvement would be to provide separate .flake8
```

Comment:
This isn't realistically possible since we have Python 2 and 3 code in the same file with `if sys.version_info` checks. I think we can just remove these two lines.